### PR TITLE
editor sidebar changes [stage-3]

### PR DESCRIPTION
**Pull request by ghislaineguerin**
@alex-deaconu here's the sidebar changes. thanks

### CODE REVIEW

**Comment by ezequielc** [comment]
@alex-deaconu I fixed the functionalities and updated E2E tests. Please review. Thanks!

File: web/partials/editor/placeholder-playlist.html [hunk 1]
```diff
@@ -1,32 +1,62 @@
...
+
+    <div ng-hide="factory.getItems().length">
+      <p class="text-center text-muted" translate>editor-app.playlistItem.no-playlistItem</p>
+    </div>
+
+
+  </div><!--playlist-->
+</div><!--editor-playlist-->
+
+<div class="content-box half-top hidden">
```
Comment (alex-deaconu) [inline web/partials/editor/placeholder-playlist.html]: @ghislaineguerin @ezequielc this is a duplicate Playlist Items list (with a table) that's `hidden`. I believe it should be removed. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu I removed the duplicate Playlist Items table and fixed i18n strings. This is ready for review. Thanks!

**Comment by alex-deaconu** [comment]
:+1: . Thanks for fixing.

**Pull request merged**